api/v1beta2/moduletemplate_types.go

```diff
@@ -69,6 +69,13 @@ func (d *Descriptor) DeepCopyObject() machineryruntime.Object {
 	return &Descriptor{ComponentDescriptor: d.Copy()}
 }
 
+type MediaTypeMetadata string
+
+const (
+	MediaTypeFile MediaTypeMetadata = "application/octet-stream"
```

Comment:
These two mediatype you'd better import directly from ocm dependencies, to keep consistency.
it's here https://github.com/open-component-model/ocm/blob/releases/v0.12.0/pkg/mime/types.go#L11-L24